packages/messages/src/browser/notification-center-component.tsx

```diff
@@ -67,13 +65,11 @@ export class NotificationCenterComponent extends React.Component<NotificationCen
                         </ul>
                     </div>
                 </div>
-                <PerfectScrollbar className='theia-notification-list-scroll-container'>
```

Comment:
`theia-notification-list-scroll-container` should be recreated without `ps` then.